schema/test/unit/computed_columns/form_change_payment_percentage_test.sql

```diff
@@ -93,6 +111,34 @@ select is(
   'should return 200 if the form change does not have adjustedEmissionsIntensityPerformance and has all information to calculate ei performance'
 );
 
+-- should return 100 if payment percentage is very large (over 100%)
+select is(
+  (
+    with record as (
+    select row(form_change.*)::cif.form_change
+    from cif.form_change where id=4
+    ) select cif.form_change_payment_percentage((select * from record))
+  ),
+  (
+    100::numeric
+  ),
+  'payments percentage is 100 from the formula: greatest(least(100 - ((-1.5) x 500 + 145), 100), 0) = 100'
```

Comment:
Nit: when giving the formula it might be better to give it in a more readable format than SQL (I've never used `least`/`greatest` before so I had to go look up the parameters and their order). The expected result is already explained on line 114 so I don't think this is a necessary change, but something like  `100 – ((-1.5) x GHG Emission Intensity Performance + 145), values capped at 0 and 100` or whatever in the test output would be clearer for me

Reply:
Good call - will update this :)